crates/sui-types/src/move_package.rs

```diff
@@ -50,6 +50,13 @@ pub struct FnInfoKey {
 /// A map from function info keys to function info
 pub type FnInfoMap = BTreeMap<FnInfoKey, FnInfo>;
 
+/// Identifies a struct and the module it was defined in
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Deserialize, Serialize, Hash)]
+pub struct ModuleStruct {
+    pub module_name: Identifier,
```

Comment:
I notice that `module_map` in `MovePackage` refers to module names using `String`.  Should we be doing the same thing here, or vice-versa?